ElectronicObserver/Window/Wpf/Log/LogView.xaml

```diff
@@ -16,19 +16,105 @@
 	TextElement.Foreground="{Binding Foreground, RelativeSource={RelativeSource AncestorType=Window}}"
 	mc:Ignorable="d"
 	>
+	<UserControl.Resources>
+		<Style x:Key="DefaultListBoxItemStyle" TargetType="ListBoxItem">
```

Comment:
Was this copied from ModernWPF? If yes, then rename the key to DefaultListBoxItemStyle**EO** and use `BasedOn={StaticResource DefaultListBoxItemStyle}`. After that's done, you can delete all properties aside from Template, since they will be inherited from the original ModernWPF style.